content/go/concepts/math-functions/terms/acos/acos.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: 'Acos()'
+Description: 'Returns the inverse of the cosine value of a number.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Math'
+  - 'Trigonometry'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Acos()`** function returns the cos inverse value of the argument (in radians). The same value is single and of the type float64.
+
+## Syntax
+
+```pseudo
+func Acos (x float64) float64
```

Comment:
```suggestion
result := math.Acos(x) 
```
We generally show the syntax of the function in use rather than showing the function signature.